application/auth/google/endpoint_legacy.go

```diff
@@ -0,0 +1,13 @@
+// +build !go1.9
+
+package google
+
+import (
+	"golang.org/x/oauth2"
+)
+
+// Endpoint is Google's OAuth 2.0 endpoint.
+var Endpoint = oauth2.Endpoint{
```

Comment:
How does this not conflict with the same variable in `endpoint.go`?

Reply:
I just borrowed those files, but it has to do with the top line `// +build !go1.9`

Reply:
Oh, I see. Thanks.